tests/libvmi/status.go

```diff
@@ -38,3 +38,15 @@ func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace st
 
 	return controlledPod
 }
+
+func GetInterfaceStatusByName(vmi *v1.VirtualMachineInstance) (map[string]v1.VirtualMachineInstanceNetworkInterface, error) {
+	if vmi == nil {
+		return nil, fmt.Errorf("failed GetInterfaceStatusByName vmi is nil")
+	}
```

Comment:
The error is of the caller and not the operation of this function.
I would recommend to avoid this defensive pattern in general.

Reply:
Ok, removing the check.